src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,12 @@
 package core.basesyntax;
 
+import java.util.Random;
+
 public class ColorSupplier {
+    private Random random = new Random();
+
     public String getRandomColor() {
-        return null;
+        int colorIndex = new Random().nextInt(DifferentColors.values().length);
```

Comment:
There's no need to create a new instance of Random every time the method is called. You already have an instance variable 'random' that you can use.